mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -1720,6 +1720,9 @@ pub struct MakerOrder {
     /// A custom priv key for more privacy to prevent linking orders of the same node between each other
     /// Commonly used with privacy coins (ARRR, ZCash, etc.)
     p2p_privkey: Option<SerializableSecp256k1Keypair>,
+    // Indicates whether the maker order is offline hence, it's not eligible for order matching.
+    #[serde(default)]
+    is_offline: bool,
```

Comment:
I would prefer more general solution here than putting very specific status as a field, e.g.,:

```
enum OrderStatus {
   Active,
   Passive,
   OnGoing,
   Unprocessable/ClientsAreOffline
}
```

and use it as `status` field in order types.